scripts/fetch/site_stats.R

```diff
@@ -0,0 +1,28 @@
+fetchTimestamp.site_stats <- vizlab::alwaysCurrent
+
+#' @title Get the discharge quantiles for each dv gage
+#' 
+#' @param viz a vizlab object that depends on a vector of sites
+fetch.site_stats <- function(viz){
+  library(dataRetrieval)
+  
+  deps <- readDepends(viz)
+  checkRequired(deps, "sites")
+  sites <- deps[["sites"]]
+  
+  req_bks <- seq(1, length(sites),by=10)
+  stat_data <- data.frame()
+  for(i in req_bks) {
+    get_sites <- sites[i:(i+9)]
+    current_sites <- suppressWarnings(
+      readNWISstat(siteNumbers = get_sites,
+                   parameterCd = "00060", 
+                   statReportType="daily",
+                   statType=c("P05", "P10", "P20", "P25", "P50", 
```

Comment:
Are we using mean? 
Also, we probably want to move these percentiles into a global param, like:
```yml
parameter:
  - id: discharge_percentiles
    percentiles: ["05","10","20","25","50","75","80","90","95"]
```

Reply:
No, doesn't look like mean is being used. Just leftover from previous code. I like parameterization idea - incorporating that now.